src/Model/Config/Communication.php

```diff
@@ -118,7 +117,7 @@ protected function getChannel(string $langAbbr): string
         $channels = $this->getConfig('ffChannel');
 
         if (!isset($channels[$langAbbr])) {
-            throw new RuntimeException("No channel for used language: $langAbbr");
```

Comment:
Why you keep deleting that exception? think it is the second time. Is added for purpose, telling user that application will not work if module is not configured.
What is the benefit of having module enabled but not configured? Search, suggest, navigation will not work.

Reply:
exception has been restored